app/styles/components/_mega-menu.scss

```diff
@@ -218,23 +218,11 @@
   }
 
   .mega-menu__mobile-categories {
-    overflow-x: scroll;
     flex-wrap: nowrap !important;
-    white-space: nowrap;
     position: sticky;
     top: 0;
     background-color: _color(grey, 700) !important;
 
-    -ms-overflow-style: none;  /* Internet Explorer 10+ */
-
-    /* stylelint-disable */
-    scrollbar-width: none;  /* Firefox */
-    /* stylelint-enable */
-
-    &::-webkit-scrollbar {
-      display: none !important;
-    }
-
```

Comment:
Vyhadzujem z megamenu, a robim z toho univerzalnu utility classu, lebo to uz ideme pouzivat na dalsich miestach v listingu kategorii napr.
Ciel: horizontalny / vertikalny scroll na mobile pri ktorom sa nezobrazi scrollbar.